docs/release_notes/0.9.0.md

```diff
@@ -0,0 +1,12 @@
+# Release 0.9.0
+
+## Features
+
+- Mark Kubernetes 1.11 as [deprecated](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html) (#1384)
```

Comment:
Not important, but If you check the release notes for prior versions, we have been following a convention of not capitalising the first word.

```suggestion
- mark Kubernetes 1.11 as [deprecated](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html) (#1384)
```

Reply:
FWIW, I did see mixed casing in `0.8.0`, hence capitalised all first letters.
I'll lowercase all then.

Reply:
Done.